functions/captureChannelWithBot.protected.ts

```diff
@@ -44,8 +44,8 @@ export type Body = {
   message: string; // (in Studio Flow, trigger.message.Body) The triggering message
   fromServiceUser: string; // (in Studio Flow, trigger.message.From) The service user unique name
   studioFlowSid: string; // (in Studio Flow, flow.flow_sid) The Studio Flow sid. Needed to trigger an API type execution once the channel is released.
-  language: string;
-  type: BotType;
+  language: string; // (in Studio Flow, {{trigger.message.ChannelAttributes.pre_engagement_data.language | default: 'en-US'}} )
```

Comment:
Is `{{trigger.message.ChannelAttributes.pre_engagement_data.language | default: 'en-US'}}` valid Studio Flow syntax? Cool!